### PR TITLE
fix storage driver options in man page

### PR DESCRIPTION
**Pull request by zhuguihua**
Signed-off-by: Zhu Guihua zhugh.fnst@cn.fujitsu.com

### CODE REVIEW

File: man/docker-daemon.8.md [hunk 1]
```diff
@@ -244,9 +244,10 @@ internals) to create writable containers from images.  Many of these
 backends use operating system level technologies and can be
 configured.
 
-Specify options to the storage backend with **--storage-opt** flags. The only
-backend that currently takes options is *devicemapper*. Therefore use these
-flags with **-s=**devicemapper.
+Specify options to the storage backend with **--storage-opt** flags. The
+backends that currently takes options  are *devicemapper* and *zfs*.
```
Comment (thaJeztah) [inline man/docker-daemon.8.md]: nit: "takes" -> "take"

File: man/docker-daemon.8.md [hunk 2]
```diff
@@ -464,6 +465,16 @@ this topic, see
...
 
+Here is supported options of *zfs*:
+
+### zfs.fsname
+
+Set zfs filesystem under which docker will create its own datasets.
+By default docker will pick up the zfs filesystem where docker graph
+(`/var/lib/docker`) is located.
+
+Example use: `docker daemon -s zfs --storage-opt zfs.fsname=zroot/docker`
```
Comment (thaJeztah) [inline man/docker-daemon.8.md]: should `zroot/docker` be an absolute path? (`/zroot/docker`) - I don't know, haven't used ZFS myself

File: man/docker-daemon.8.md [hunk 3]
```diff
@@ -464,6 +465,16 @@ this topic, see
 Otherwise, set this flag for migrating existing Docker daemons to a
 daemon with a supported environment.
 
+Here is supported options of *zfs*:
```
Comment (thaJeztah) [inline man/docker-daemon.8.md]: I think we should make this a heading, for example:

```Markdown
## ZFS options
```

And do the same for [line 262](https://github.com/zhuguihua/docker/blame/fix_storage_driver_options_in_man_page/man/docker-daemon.8.md#L262); change "Here is the list of *devicemapper* options:" to

```Markdown
## Devicemapper options
```

**Comment by thaJeztah** [comment]
Thanks @zhuguihua, I left some suggestions.

File: man/docker-daemon.8.md (hunk 2 above)
Comment (zhuguihua) [inline man/docker-daemon.8.md]: I think zroot/docker is right.

**Comment by zhuguihua** [comment]
@thaJeztah Thanks for your suggestions. I applied them.

**Comment by thaJeztah** [comment]
@zhuguihua thanks!

LGTM

File: man/docker-daemon.8.md [hunk 4]
```diff
@@ -244,9 +244,10 @@ internals) to create writable containers from images.  Many of these
 backends use operating system level technologies and can be
 configured.
 
-Specify options to the storage backend with **--storage-opt** flags. The only
-backend that currently takes options is *devicemapper*. Therefore use these
-flags with **-s=**devicemapper.
+Specify options to the storage backend with **--storage-opt** flags. The
+backends that currently take options  are *devicemapper* and *zfs*.
```
Comment (vdemeester) [inline man/docker-daemon.8.md]: A *nit* but, there is one space too much `options  are`

**Comment by zhuguihua** [comment]
@vdemeester Sorry for my mistake. I fixed that.

**Comment by vdemeester** [comment]
@zhuguihua cool, thanks :wink: 

LGTM 🐴

**Pull request merged**